gapic/samplegen/samplegen.py

```diff
@@ -92,6 +117,20 @@ class InvalidRequestSetup(SampleError):
     pass
 
 
+def resp_to_response(s):
```

Comment:
I do not love making this a new filter in the Jinja environment; `|replace('$resp', 'response')` does the same thing.

If we need a "single home" for this logic, I would suggest naming it something more generic (e.g. `coerce_variable_name`) so that it can accommodate future massaging that may come up.

Reply:
@vchudnov-g commented earlier that he was not fond of the repeated `|replace('$resp', 'response')`, so quick and dirty filter it was.
`coerce_variable_name` is more helpful from a slightly more abstract viewpoint but less descriptive of what is actually happening. The ideal solution would be to somehow use an in-template macro as a filter and avoid messing with the environments.

Reply:
Going with `coerce_variable_name` for the time being.

Reply:
sg